api_tests/e2e/rfc003/btc_eth/happy.js

```diff
@@ -151,7 +151,8 @@ describe("RFC003: Bitcoin for Ether", () => {
             "amount",
             "network"
         );
-        await alice.do(alice_fund_action);
+        let resp = await alice.do(alice_fund_action);
```

Comment:
❌ no need to assign a value if it's not used.
Also confusing as you know have a `resp` and `res` variable.